addons/website_sale_delivery/static/src/js/website_sale_delivery.js

```diff
@@ -5,14 +5,18 @@ var core = require('web.core');
 var publicWidget = require('web.public.widget');
 
 var _t = core._t;
+var qweb = core.qweb;
```

Comment:
Please follow current guidelines for new code (even if surrounded by legacy code that does not)
```suggestion
const qweb = core.qweb;
```